docs/api.md

```diff
@@ -781,7 +705,7 @@ This adds a filter `shorten` which returns the first `count`
 characters in a string, with `count` defaulting to 5. Here is how it
 is used:
 
-```jinja
+```html
```

Comment:
I'd keep this `jinja` and change it in the formatter (maybe we can also have the formatter accept `nunjucks` and `njk`).